pkgs/applications/editors/emacs/elisp-packages/update-from-overlay.nix3.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env nix
+#! nix shell nixpkgs#bash nixpkgs#curl nixpkgs#nix nixpkgs#coreutils --command bash
+set -euxo pipefail
+
+# Since Nix3 is not in the default installation, the original
+# ./update-from-overlay can't be updated. Instead of it, let's create a wrapper!
```

Comment:
Based on the shebang above, `flake` is more accurate than `Nix3` here because you can still use `<nixpkgs>` in a Nix3 CLI.

I notice that there is `export NIXPKGS_ALLOW_BROKEN=1` in [update-from-overlay](https://github.com/NixOS/nixpkgs/blob/39341b72a0d6df5c23f4185ca6b2d06179364af5/pkgs/applications/editors/emacs/elisp-packages/update-from-overlay#L8).  I do not think that works in flake.

Reply:
I will rename it, then.